devices/tuya.js

```diff
@@ -2496,6 +2496,51 @@ module.exports = [
         },
         whiteLabel: [{vendor: 'Hiking', model: 'DDS238-2'}, {vendor: 'TuYa', model: 'RC-MCB'}],
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_lsanae15']),
+        model: 'TS0601_din_2',
+        vendor: 'TuYa',
+        description: 'Zigbee DIN energy meter',
+        fromZigbee: [tuya.fzDataPoints],
+        toZigbee: [tuya.tzDataPoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [tuya.exposes.switch(), e.energy(), e.power(), e.voltage(), e.current(),
+            exposes.enum('fault', ea.STATE, ['clear', 'over_current_threshold', 'over_power_threshold',
+                'over_voltage threshold', 'wrong_frequency_threshold']).withDescription('Fault status of the device (clear = nothing)'),
+            exposes.binary('threshold_1_state', ea.STATE, 'ON', 'OFF')
+                .withDescription('OFF - alarm only, ON - relay will be off when threshold reached'),
+            exposes.text('threshold_1_description', ea.STATE), // TODO: what is this?
+            exposes.numeric('threshold_1_value', ea.STATE)
+                .withDescription('Can be in Volt or Ampere depending on threshold setting. Setup the value on the device'),
+            exposes.binary('threshold_2_state', ea.STATE, 'ON', 'OFF')
+                .withDescription('OFF - alarm only, ON - relay will be off when threshold reached'),
+            exposes.text('threshold_2_description', ea.STATE),
+            exposes.numeric('threshold_2_value', ea.STATE)
+                .withDescription('Setup value on the device'),
+            exposes.binary('clear_fault', ea.STATE_SET, 'ON', 'OFF')
+                .withDescription('Turn ON to clear last the fault'),
+            exposes.text('meterid', ea.STATE).withDescription('Meter ID'), // TODO: update description
```

Comment:
Can you add a better description here?

Reply:
It's just ID of device. Did it
![telegram-cloud-photo-size-2-5449377471087623675-y](https://user-images.githubusercontent.com/81035252/202006549-c51605b6-925a-4a62-854d-0d58a9ff439f.jpg)